rc.js

```diff
@@ -16,30 +19,34 @@ if (process.env.npm_config_argv) {
   } catch (e) { }
 }
 
-var npmconfigs = ['proxy', 'https-proxy', 'local-address', 'target', 'runtime', 'platform']
-for (var j = 0; j < npmconfigs.length; ++j) {
-  var envname = 'npm_config_' + npmconfigs[j].replace('-', '_')
-  if (process.env[envname]) {
-    process.argv.push('--' + npmconfigs[j])
-    process.argv.push(process.env[envname])
-  }
-}
-
+/**
+ * Get the configuration
+ *
+ * Values are fetch from the CLI arguments, the `pkg` object (probably from the
+`* package.json` file), the environment variables set by the `npm` instance that
+ * executed `prebuild-install`, and finally some safe default values from the
+ * environment variables or the current Node.js instance
+ *
+ * @param {object} pkg
+ *
+ * @return {object} the aggregated configuration
+ */
```

Comment:
What do you mean, the jsdoc? Yeah, I forgot to document the fields of `pkg`... I can be able to add them.